src/hotspot/share/jeandle/jeandleRuntimeRoutine.cpp

```diff
@@ -106,3 +106,11 @@ JRT_ENTRY(address, JeandleRuntimeRoutine::search_landingpad(JavaThread* current)
 
   return nm->code_begin() + handler_pc_offset;
 JRT_END
+
+//=============================================================================
+//                      Array Allocation Routines
+//=============================================================================
```

Comment:
```suggestion
// Array allocation
```
In fact, this is also belongs to `Jeandle Runtime C/C++ Routines`